react/features/base/conference/reducer.js

```diff
@@ -183,19 +183,31 @@ function _conferenceJoined(state, { conference }) {
  * reduction of the specified action.
  */
 function _conferenceLeft(state, { conference }) {
-    if (state.conference !== conference) {
-        return state;
+    let output = state;
+
+    if (state.authRequired === conference) {
+        output = set(output, 'authRequired', undefined);
+    }
+    if (state.conference === conference) {
+        output = assign(output, {
+            conference: undefined,
+            joining: undefined,
+            leaving: undefined
+        });
+    }
+    if (state.passwordRequired === conference) {
+        // Note that in case the conference was joined those fields have been
+        // cleared already, so this step needs to be done only if the room
+        // unlock operation has been canceled and that's why it's not done in
+        // the 'state.conference' condition above.
+        output = assign(output, {
+            locked: undefined,
+            password: undefined,
```

Comment:
I don't think the comment and the source code are 100% correct. `locked` and `password` need to be cleared here when we're talking about a conference that's been `LOCKED_REMOTELY`. But we need to clear in the condition `state.conference` above as well when we're talking about a conference that's been `LOCKED_LOCALLY`. I'll take care of it.